docker-compose.yml

```diff
@@ -18,4 +18,6 @@ services:
       DATABASE_URL: "ecto://postgres:postgres@db/postgres"
   db:
     image: postgres:alpine
-    container_name: papercups
```

Comment:
I don't know what this was doing but I couldn't get the docker-compose to work, complained about not having a user setup.

Reply:
cc @cheeseblubber do you know anything about this?